source/numerics.tex

```diff
@@ -1786,6 +1786,30 @@
 \indextext{requirements!uniform random bit generator|)}%
 \indextext{uniform random bit generator!requirements|)}
 
```

Comment:
Not done: "[Editor’s note: Relocate "Header /tcode<random> synopsis" [rand.synopsis] before 29.6.1 "Requirements"
[rand.req]]"

Reply:
I must have ignored this directive upon finding no header named `/tcode<random>` in the library ;)